### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by miltonHenschel**
- [ ] Modified all sections of the mobile version to suit the desktop version of `1440px` screen size.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -385,15 +419,178 @@ hr {
...
+  }
+
+  .headline div {
+    order: 2;
+    width: 49%;
+  }
+
+  .headline h2 {
+    padding: 1rem 0;
+  }
```
Comment (jerryowusu) [inline css/styles.css]: There are some slight differences between your header and that of the figma design as shown below :point_down: . Kindly check the style topography (font-size, font-weight, line-height, etc) and the dimensions and fix accordingly. 

_Please note that the images shown on this and other instances were taken from certain screen widths of your desktop version and varies with increasing/decreasing screen sizes. However at all screen sizes of your website, it should be similar just as depicted in the actual design_


 Your design            |  The design template
:-------------------------:|:-------------------------:
Screenshot from 2022-11-06 06-13-30  |  Screenshot from 2022-11-06 06-13-54

File: css/styles.css [hunk 2]
```diff
@@ -385,15 +419,178 @@ hr {
...
+  #projects-title {
+    padding: 10rem 2rem 10rem;
+    font-size: 2.5rem;
+  }
+
+  #flex-projects {
+    display: flex;
+    justify-content: space-around;
+    flex-direction: row;
+  }
```
Comment (jerryowusu) [inline css/styles.css]: - [ ] Please make sure that the featured project looks similar to the design as much as possible in all screen sizes. Curently the image takes a greater portion of the width :point_down: . This should not be so. Ensure that the image always takes 60% of the page's width.



 Your design            |  The design template
:-------------------------:|:-------------------------:
Screenshot from 2022-11-06 07-15-25  |  Screenshot from 2022-11-06 06-14-12

File: css/styles.css [hunk 3]
```diff
@@ -385,15 +419,178 @@ hr {
...
+
+  #flex-projects h2 {
+    font-size: 1.5rem;
+    padding-top: 1rem;
+  }
+
+  #flex-projects div p {
+    font-size: 1.25rem;
+  }
```
Comment (jerryowusu) [inline css/styles.css]: Please check and make sure that the project section uses the same fonts, alignments, paddings and margins like the figma design. Kindly check and rectify this anomaly. :arrow_down: 


 Your design            |  The design template
:-------------------------:|:-------------------------:
Screenshot from 2022-11-06 07-12-19  |  Screenshot from 2022-11-06 06-15-08

File: css/styles.css [hunk 4]
```diff
@@ -385,15 +419,178 @@ hr {
...
+    flex-direction: column;
+    align-items: normal;
+  }
+
+  /* Languages */
+  #about-me-lang {
+    display: flex;
+    gap: 2rem;
+    justify-content: space-evenly;
+  }
```
Comment (jerryowusu) [inline css/styles.css]: - [ ] There are still some work to be done on the about me section in order to match the figma design. Kindly check and update the color, font-size, font-weight, font-style, dimensions, etc.
- [ ] The background image is also missing. Kindly add that :arrow_down_small: 


 Your design            |  The design template
:-------------------------:|:-------------------------:
Screenshot from 2022-11-06 06-16-20  |  Screenshot from 2022-11-06 06-16-13

File: css/styles.css [hunk 5]
```diff
@@ -385,15 +419,178 @@ hr {
...
+
+  #form h2 {
+    text-align: left;
+    width: 39%;
+  }
+
+  #contact-form {
+    display: unset;
+    width: 59%;
+  }
```
Comment (jerryowusu) [inline css/styles.css]: You are using a different font-size and color compared to the design. Also the width, margins, height, etc are slightly different :point_down:  Kindly check and ensure that all are similar to the actual design.


 Your design            |  The design template
:-------------------------:|:-------------------------:
Screenshot from 2022-11-06 06-16-58  |  Screenshot from 2022-11-06 06-16-45

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @miltonHenschel :wave: 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, I've found some room for improvement that can help you accomplish a more readable/clean code and make your project look professional :sunglasses:

so close


## Highlights
Nice documentation :heavy_check_mark: 
Good commit messages :heavy_check_mark: 
Followed GitHub flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -235,24 +241,30 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
       <form id="contact-form" action="https://formspree.io/f/mrgdolqb" method="POST">
         <input type="text" autocomplete="off" name="name" id="name" placeholder="Full name" maxlength="30" required />
 
+        <input type="text" autocomplete="off" name="firstname" id="firstname" placeholder="First name" maxlength="30" required />
+
+        <input type="text" autocomplete="off" name="lastname" id="lastname" placeholder="Last name" maxlength="30" required />
+
```
Comment (leonardodiasb) [inline index.html]: - [ ] Your form is very similar to the Figma template, but there is an important note in the project requirement mentioning that the part of the name input fields should be ignored in the desktop version for templates that has 2 input fields, one for the first name and other for the last name, and use only one input field for the full name in both versions(desktop and mobile). Please remove the input fields you added and use only one input field for the full name as the project requirement asks for.
 - 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @miltonHenschel, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Correct use of Github flow. ✔️ 
 - Professional PR message. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (miltonHenschel) [reply index.html]: @leonardodiasb thanks, I completely forgot that remark.

**Review by christianbiring1 — APPROVED** [review]
Hi @miltonHenschel , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
Iapproveokay

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Good commit message.
## Optional suggestions
- [ ] `YOU MAY ALSO CONSIDER REDUCING THE SPACING(MARGIN, PADDING) OR YOUR FORM INPUTS, THEY APPEAR TO BE TOO BIG.` :smile: 
image

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**